packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -573,4 +603,18 @@ class WKWebView extends NSObject {
   Future<Object?> evaluateJavaScript(String javaScriptString) {
     throw UnimplementedError();
   }
+
+  /// The view’s background color.
+  ///
+  /// The default value is null, which results in a transparent background color.
```

Comment:
Really? I thought a WKWebView's default background color was white.

Reply:
Ah, this is from the `UIView` docs actually. I suppose this may not be true for the subclasses. I added the `UIView` class, so it should be accurate now.